.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: build
+
+on:
+  push:
+    branches:
+      - "master"
+    tags:
+      - "v*"
+  pull_request:
```

Comment:
Is this right? Should there not be a value?

Reply:
It is from their docs so I guess it's good: https://goreleaser.com/ci/actions/